EIPS/eip_195_pure_call.md

```diff
@@ -0,0 +1,23 @@
+### Parameters
+
+* BASE_GAS_COST: 80
+* CODE_BYTES_PER_GAS: 15
```

Comment:
Assuming we want to switch to EVM 1.5 - is that really the amount of gas we want to charge? Or shall we completely revise the scheme with 1.5?